portfolio.html

```diff
@@ -13,31 +13,37 @@
 	<body>		
 		<nav class="nav-bar">
 			<a class="navigation-links" href="#home">home</a> 
-<!--need to root home button to different part of page-->
 			<a class="navigation-links" href="#about-me">about me</a>
 			<button class="cv-button">download my CV!</button>
 			<a class="navigation-links" href="#portfolio">portfolio</a>
 			<a class="navigation-links" href="#contact-me">contact me</a>	
 		</nav>	
 	
-		<section>
-			<div class="welcome-banner">
-				<h1 class="hi-there-welcome-text">
+		<section class="welcome-banner">
+			<div>
+				<h1 class="small-title">
+					<img class="sparkle-inline" src="sparkles.gif" alt="sparkle">
 					<a id="home">hi there</a>
 				</h1>
-				<h2 class="welcome-text">
+			</div>
+			<div>
+				<h2 class="large-title">
```

Comment:
If you switch the section name to header then you could select this with header h2 and not need the class name